### PR TITLE
feat(UI): simplify code

### PR DESCRIPTION
**Pull request by saghul**
Avoid branching on the same condition and remove unneeded showToolbar call on
filmstrip only mode.

### CODE REVIEW

**Comment by saghul** [comment]
The test failure is unrelated.

File: modules/UI/UI.js [hunk 1]
```diff
@@ -340,30 +340,30 @@ UI.start = function() {
...
 
-    // TODO: remove this class once the old toolbar has been removed. This class
-    // is set so that any CSS changes needed to adjust elements outside of the
-    // new toolbar can be scoped to just the app with the new toolbar enabled.
-    if (!interfaceConfig.filmStripOnly) {
+        // TODO: remove this class once the old toolbar has been removed. This
+        // class is set so that any CSS changes needed to adjust elements
+        // outside of the new toolbar can be scoped to just the app with the new
+        // toolbar enabled.
         $('body').addClass('use-new-toolbox');
```
Comment (virtuacoplenny) [inline modules/UI/UI.js]: Oh, I should change that classname...

File: modules/UI/UI.js [hunk 2]
```diff
@@ -340,30 +340,30 @@ UI.start = function() {
...
-    }
 
-    if (interfaceConfig.VERTICAL_FILMSTRIP) {
-        $('body').addClass('vertical-filmstrip');
-    }
+        // XXX: We only do this when not in film strip only mode, since
+        // vertical toolbars are not implemented there.
+        if (interfaceConfig.VERTICAL_FILMSTRIP) {
+            $('body').addClass('vertical-filmstrip');
+        }
```
Comment (virtuacoplenny) [inline modules/UI/UI.js]: So this class no longer gets added when in vertical filmstrip only mode? How does that look? The idea is that even though vertical filmstrip only might not have a decent experience that it still functions. Just like horizontal filmstrip with the new toolbar...it works but it sucks but it works, and it works without much extra effort code-wise.

Comment (saghul) [reply modules/UI/UI.js]: What I did there is essentially not add it, yes. For all intents and purposes we only seem to support horizontal filmstrip mode.

I wonder if we should remove the setting and make it intrinsic: normal view uses a vertical filmstrip and filmstriponly uses a horizontal one.

Comment (virtuacoplenny) [reply modules/UI/UI.js]: What I'm saying is it's somewhat supported but clearly not a great UX; it's not the prettiest but it still works like the regular vertical filmstrip. I recall being explicitly told while implementing vertical filmstrip to make vertical filmstrip display in filmstrip only mode. That it's not optimized or pretty is not the same as not being supported. As for forcing orientation, I commented in the jira ticket you filed and I'll repeat here: it was actually easier for now to add 5-10 lines of CSS to support both filmstrip orientations than to refactor web right now to force a certain filmstrip orientation. When going into horizontal toolbar it was discussed to force the filmstrip orientations, but there are a lot of tacked on conditionals spread across web that made that effort hairier than I cared for.

Comment (virtuacoplenny) [reply modules/UI/UI.js]: So I guess what I'm getting at is removing the class would be something I'm okay with if it's a more concerted effort to kill off support for vertical filmstrip in filmstrip only mode. Removal of the styling itself I would consider insufficient.

**Comment by saghul** [comment]
@virtuacoplenny After taking a second look after a break, you're right. I restored setting the vertical filmstrip class. PTAL, and thanks for your patience here!

File: modules/UI/UI.js (hunk 1 above)
Comment (virtuacoplenny) [reply modules/UI/UI.js]: Man, I really should change that classname...

**Pull request merged**